romdisk/uart.asm

```diff
@@ -115,7 +117,9 @@ uartsnd_main:
         dec bc  ; Do not count the command itself
         ld a, b
         or c
+        jr z, _uartsnd_usage
         ret z
+
         inc hl
         inc hl
         ld a, (hl)
```

Comment:
To optimize the code here, read directly in BC:
```
ld c, (hl)
inc hl
ld b, (hl)
```

Reply:
#fixed